app/assets/javascripts/tests/helpers.js

```diff
@@ -8,6 +8,7 @@ export const mock = name => {
   return {
     [name]: (arg1, arg2, arg3) => { calls1.push(arg1); calls2.push(arg2); calls3.push(arg3); },
     wasCalled: () => calls1.length === 1,
+    wasCalled3Times: () => calls1.length ===3,
```

Comment:
Have you considered refactoring `wasCalled` to be used like `wasCalled({ times: 3 })` or introducing `wasCalledNTimes(3)`?